docs/user-guide/command-line-interface.md

```diff
@@ -27,56 +27,53 @@ Please note that when passing a glob as a parameter, it will be expanded by your
 The command line utility has several options. You can view the options by running `eslint -h`.
 
 ```text
+eslint [options] file.js [file.js] [dir]
```

Comment:
Nit: Would it be worth using `file1.js [file2.js]` to clarify that multiple different files can be specified?

Also, the first file isn't required either (since you can use `eslint dir/` directly, so maybe it would be better as `eslint [options] [file1.js] [file2.js] [dir]` or something like that.